go/tools/gopackagesdriver/bazel_json_builder.go

```diff
@@ -17,72 +17,102 @@ package main
 import (
 	"context"
 	"fmt"
+	"path/filepath"
 	"strings"
 )
 
 type BazelJSONBuilder struct {
-	bazel      *Bazel
-	query      string
-	tagFilters string
-	targets    []string
+	bazel    *Bazel
+	requests []string
 }
 
 const (
 	OutputGroupDriverJSONFile = "go_pkg_driver_json_file"
 	OutputGroupStdLibJSONFile = "go_pkg_driver_stdlib_json_file"
 	OutputGroupExportFile     = "go_pkg_driver_export_file"
+	wildcardQuery             = `kind("go_library", //...)`
 )
 
-func NewBazelJSONBuilder(bazel *Bazel, query, tagFilters string, targets []string) (*BazelJSONBuilder, error) {
+func (b *BazelJSONBuilder) fileQuery(filename string) string {
+	if filepath.IsAbs(filename) {
+		fp, _ := filepath.Rel(b.bazel.WorkspaceRoot(), filename)
+		filename = fp
+	}
+	return fmt.Sprintf(`some(kind("go_library", same_pkg_direct_rdeps("%s")))`, filename)
+}
+
+func (b *BazelJSONBuilder) packageQuery(importPath string) string {
+	return fmt.Sprintf(`some(kind("go_library", attr(importpath, "%s", %s)))`, importPath, bazelQueryScope)
+}
+
+func (b *BazelJSONBuilder) queryFromRequests(requests ...string) string {
+	ret := make([]string, 0, len(requests))
+	for _, request := range requests {
+		if request == "." || request == "./..." {
+			return wildcardQuery
```

Comment:
gopls sends a `./...` request to package driver when VS Code opens. This wildcardQuery will also load the universe. Maybe we should just load only `bazelQueryScope`?

```suggestion
			return fmt.Sprintf("kind(go_library, %s)", bazelQueryScope)
```

Reply:
that's a very good find.
although i'm thinking it may be best to disable `./...` altogether ?

Reply:
my reasoning being, is there a valid usage to querying `./...` with `gopls`? 
I think your change may be the most sensible one, perhaps also allowing folks to specify an empty scope to disable it altogether

Reply:
I am fine with either approach. I am not familiar with gopls enough to say whether `./...` is necessary. Disabling `./...` may make gopls faster when VS code first opens

Reply:
I think I found a good workaround: by default `./...` will return only stdlib, unless a scope is specified, in which case it will use that. Same for `importpath` queries.

Reply:
That's actually the same as ignoring `./...` because gopls sends `./... builtin` request when VS Code starts.